ios/RNFirebase/config/RNFirebaseRemoteConfig.m

```diff
@@ -49,7 +59,7 @@ @implementation RNFirebaseRemoteConfig
             (RCTPromiseRejectBlock) reject) {
     [[FIRRemoteConfig remoteConfig] fetchWithCompletionHandler:^(FIRRemoteConfigFetchStatus status, NSError *__nullable error) {
         if (error) {
-            reject(convertFIRRemoteConfigFetchStatusToNSString(status), error.localizedDescription, error);
+            reject(convertFIRRemoteConfigFetchStatusToNSString(status), convertFIRRemoteConfigFetchStatusToNSStringDescription(status), error);
         } else {
             resolve(convertFIRRemoteConfigFetchStatusToNSString(status));
```

Comment:
Could change to nil - doesn't need to resolve any sort of meaningful value really.